packages/web/src/modules/core/styles/switchStyle.ts

```diff
@@ -0,0 +1,55 @@
+import { styled } from "../../theming/themes"
+const size = 30
+const color = "#009fb7"
+
+export const LabelSwitch = styled.label`
+    position: relative;
+    display: inline-block;
+    width: ${size * 2}px;
+    height: ${size}px;
+`
+export const InputSwitch = styled.input`
+    & {
+        opacity: 0;
+        width: 0;
+        height: 0;
+    }
+    &:checked + span {
+        background-color: ${color};
+    }
+    &:checked + span:before {
+        -webkit-transform: translateX(${(size * 2 - 4) / 2}px);
+        -ms-transform: translateX(${(size * 2 - 4) / 2}px);
+        transform: translateX(${(size * 2 - 4) / 2}px);
+    }
+`
+export const SpanSwitch = styled.span`
+    & {
```

Comment:
You know you can just remove the & {, right ?

Reply:
done